pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1543,9 +1636,11 @@ func (t *Task) deleteProgressReportingCRs(client k8sclient.Client) error {
 
 	for ns, vols := range pvcMap {
 		for _, vol := range vols {
+			dvmpName := getMD5Hash(t.Owner.Name + vol.Name + ns)
+			t.Log.Info(fmt.Sprintf("DELETING stale DVMP CR [%v/%v]", dvmpName, ns))
 			err := client.Delete(context.TODO(), &migapi.DirectVolumeMigrationProgress{
 				ObjectMeta: metav1.ObjectMeta{
-					Name:      getMD5Hash(t.Owner.Name + vol.Name + ns),
+					Name:      dvmpName,
```

Comment:
Just as future implementation note, I prefer meaningful names using `generate-name` to get uniqueness. 

The reason being a user can then hopefully tell what that thing is for rather than just some random pods getting spun up.

Reply:
+1. I recall that in some places, we started with `generateName` and switched to deterministic names so that if a CREATE call failed and we had to exit the reconcile, we could re-discover anything we've successfully created so far and not end up with duplicate resources upon second reconcile attempt.

This was important when we weren't using ownerRefs, but since I think DVMP stuff uses ownerRefs we could probably switch back to using generateName in some places.

Reply:
yeah, a label and an owner ref comparison might do the trick?